owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.kt

```diff
@@ -332,6 +347,14 @@ class PreviewImageFragment : FileFragment() {
         return if (isSVGFile(file)) Color.WHITE else Color.BLACK
     }
 
+    private fun showSnackbarMessage() {
+        val snackbar = Snackbar.make(requireView(), R.string.sync_file_nothing_to_do_msg, Snackbar.LENGTH_LONG)
+        val params = snackbar.view.layoutParams as ViewGroup.MarginLayoutParams
+        params.setMargins(params.leftMargin, params.topMargin, params.rightMargin, 100)
```

Comment:
What happens if we don't set these margins for the Snackbar? Is it special about the image preview?